homeassistant/components/telegram_bot/services.yaml

```
@@ -50,6 +49,9 @@ send_message:
       example: "msg_to_edit"
       selector:
         text:
+    reply_to_message_id:
+      selector:
+        object:
```

Why is this an object?

---

I thought about using "number" but it requires min/max and while min: 1 would make sense, I would not want to define a max value for an ID field.
should it be text? - I chose selector because its the most "neutral" of all the selectors... but it also didn't feel 100% correct to me to be honest since an actual object (list) would not be a valid input.
so would you suggest number or text (single line)?

---

`min` & `max` are only required in slider mode.

```suggestion
        number:
          mode: box
```

---

hm okay but then the documentation is not clear:
https://www.home-assistant.io/docs/blueprint/selectors/#number-selector

> On the user interface, the input can either be in a slider or number mode. Both modes limit the user input by a minimum and maximum value

https://www.home-assistant.io/docs/blueprint/selectors/#min
both min/max state "Required"

---

okay thanks for clarifying - the blueprints selector documentation was the one I found when I was looked for documentation on selectors for services.yaml